lib/sign.js

```diff
@@ -24,7 +24,7 @@ export function createSignCryptosuite({mandatoryPointers = []} = {}) {
     name,
     requiredAlgorithm,
     createVerifier: _throwSignUsageError,
-    createVerifyData: _createSignData,
+    createSignData: _createSignData,
```

Comment:
I believe this was a typo, so I fixed it. let me know if it wasn't, I can revert it back.